test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -48,6 +48,27 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Tests
 
+(deftest ^:integration symlink-cadir-test
```

Comment:
This could probably be in with the unit tests now, right?

Reply:
Good point.

Reply:
Not important enough to make Travis rerun though 😉

Reply:
tooo late!